packages/core/test/lib/utils/traceData.test.ts

```diff
@@ -87,25 +76,24 @@ describe('getTracingMetaTags', () => {
             traceFlags: TRACE_FLAG_SAMPLED,
           };
         },
-        transaction: undefined,
       },
       mockedScope,
       mockedClient,
     );
 
     expect(tags).toEqual({
```

Comment:
l: due to above's suggestion
```suggestion
    expect(traceData).toEqual({
```